addons/sale/report/ir_actions_report_templates.xml

```diff
@@ -181,6 +180,11 @@
                                 </td>
                             </tr>
                             <tr t-else="" name="tr_product">
+                                <t
```

Comment:
If a section is set to hide prices, shouldn't we hide the `td_section_price` of its subsections too ? :thinking: